doc/sphinx/source/recipes/recipe_hydrology.rst

```diff
@@ -146,6 +151,20 @@ All hydrological recipes require a shapefile as an input to produce forcing data
    * method: contains
    * decomposed: true
 
+#. recipe_globwat.yml
+
+   *Required preprocessor settings:*
+
+   * start_year: 2004
+   * end_year: 2004
+   * target_file: Generated from a GlobWat input file sample
```

Comment:
Not sure I understand what this means. Can you add a more elaborate description of `target_file`?

Reply:
The target_grid is the grid of GlobWat input files. I added the description in the doc.